frontend/common/src/components/form/Input/Input.tsx

```diff
@@ -34,15 +36,25 @@ const Input: React.FunctionComponent<InputProps> = ({
   type,
   errorPosition = "bottom",
   hideOptional,
+  whitespaceTrim = true,
   ...rest
 }) => {
   const {
     register,
     formState: { errors },
+    setValue,
   } = useFormContext();
   // To grab errors in nested objects we need to use lodash's get helper.
   const error = get(errors, name)?.message;
 
+  const whitespaceTrimmer = () => {
```

Comment:
Reach should pass as `React.FocusEvent<HTMLInputElement>` to `onBlur` event handler. You should be able to access it like such so you do not need the `document.getElementById`:

```tsx
const handleBlur = (e : React.FocusEvent<HTMLInputElement>) => {
  const el = e.target;
  ... 
}
```